.readthedocs.yml

```diff
@@ -0,0 +1,5 @@
+build:
```

Comment:
This file can be removed once all of this is fixed and we can set the default language version in READTHEDOCS admin panel